lib/nodes/Func.js

```diff
@@ -101,7 +103,10 @@ class Func extends Container {
     parser.back(rightTokens);
 
     const { lastNode } = parser;
-    lastNode.isColor = colorFunctions.includes(lastNode.name.toLowerCase());
+    const lowerName = lastNode.name.toLowerCase();
+    const { nodes } = node;
+    lastNode.isColor = colorFunctions.includes(lowerName);
+    lastNode.isVar = lowerName === 'var' && nodes.length === 1 && reVar.test(nodes[0].value);
```

Comment:
This is too restrictive, does not allow for `default` fallback values to be defined

Reply:
fixed